plugins/modules/s3_bucket.py

```diff
@@ -124,12 +124,16 @@
   object_ownership:
     description:
       - Allow bucket's ownership controls.
+      - C(BucketOwnerEnforced) - ACLs are disabled and no longer affect access permissions to your
+        bucket. Requests to set or update ACLs fail. However, requests to read ACLs are supported.
+        Bucket owner has full ownership and control. Object writer no longer has full ownership and
+        control.
```

Comment:
Could you please add something like: ` C(BucketOwnerEnforced) has been added in amazon.aws version 3.2.0`